gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-6/api-creation-v4-step-6.component.html

```diff
@@ -0,0 +1,119 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card class="api-creation-v4__step">
+  <div class="api-creation-v4__step__header">
+    <div class="mat-h2 subtitle">Review your API configuration</div>
+    <p class="paragraph-light">Review your API before publishing</p>
+  </div>
+  <div class="api-creation-v4-step-6__body">
+    <!--    Step 1 -->
+    <div class="api-creation-v4-step-6__step">
+      <div class="api-creation-v4-step-6__step__info">
+        <div class="mat-h3">API metadata</div>
+        <div class="api-creation-v4-step-6__step__info__row">
+          <p class="paragraph-light">API name:</p>
```

Comment:
i think is better to use a span and remove all margin bottom with 8. then aligne each item with flex in `api-creation-v4-step-6__step__info__row`